src/python_testing/TC_OpstateCommon.py

```diff
@@ -1311,11 +1321,12 @@ async def TEST_TC_OPSTATE_BASE_2_6(self, endpoint=1):
             self.skip_step(7)
 
         sub_handler.reset()
+        self.step(8)
```

Comment:
The else portion of this if/else block should be removed since the step was moved here.

Reply:
thanks - removed. PTAL.